.eslintrc.cjs

```diff
@@ -33,9 +33,9 @@ module.exports = {
     'plugin:markdown/recommended',
     'plugin:import/errors',
     'plugin:eslint-comments/recommended',
-    'plugin:prettier/recommended',
     'plugin:unicorn/recommended',
     'plugin:promise/recommended',
+    'plugin:prettier/recommended',
```

Comment:
this turns off the conflicting `unicorn/number-literal-case`, so moving it to the end